tests/disas/load-store/x64/load_store_static_kind_i32_index_0_guard_yes_spectre_i32_access_0_offset.wat

```diff
@@ -21,33 +21,33 @@
 ;; wasm[0]::function[0]:
 ;;       pushq   %rbp
 ;;       movq    %rsp, %rbp
-;;       movl    %edx, %r11d
-;;       xorq    %r10, %r10
-;;       movq    %r11, %r9
-;;       addq    0x40(%rdi), %r9
-;;       cmpq    0x10(%rip), %r11
-;;       cmovaq  %r10, %r9
-;;       movl    %ecx, (%r9)
+;;       movl    %edx, %esi
+;;       xorq    %r11, %r11
+;;       movq    %rsi, %r10
+;;       addq    0x40(%rdi), %r10
+;;       cmpq    0x11(%rip), %rsi
+;;       cmovaq  %r11, %r10
+;;       movl    %ecx, (%r10)
 ;;       movq    %rbp, %rsp
 ;;       popq    %rbp
 ;;       retq
-;;   24: addb    %al, (%rax)
-;;   26: addb    %al, (%rax)
-;;   28: cld
+;;   23: addb    %al, (%rax)
+;;   25: addb    %al, (%rax)
+;;   27: addb    %bh, %ah
```

Comment:
I do have some questions about the offset changes here (hence the draft status): here, for example, we've managed to shave a byte off our encodings somehow...